packaging/test_irods_client_rest_cpp.py

```diff
@@ -190,6 +191,98 @@ def test_put_configuration(self):
         os.remove(file1)
         os.remove(file2)
 
+    def test_meta_data_object(self):
```

Comment:
These tests do not verify that the metadata operation was successful.

The tests need to show that metadata was attached to and removed from the target object.